docs/ReleaseNotes.md

```diff
@@ -41,6 +41,7 @@ The Guava dependency version has been updated to 31.1. Projects may need to chec
 
 ### 3.3.365.0
 
+* **Bug fix** Don't write to disabled Synthetic indexes [(Issue #2099)](https://github.com/FoundationDB/fdb-record-layer/issues/2099)
```

Comment:
This should go in the commented out section above rather than here (given that it is _not_ a feature of 3.3.365.0)